app/models/finance/schedule.rb

```diff
@@ -4,7 +4,15 @@ class Finance::Schedule < ApplicationRecord
   has_many :milestones, -> { order(milestone_date: :asc) }
   has_many :participant_profiles
 
-  def self.default
+  def self.default_ecf
```

Comment:
If these are going to be different between ecf and npq, that sounds like we should start namespacing more. Thoughts?

Reply:
Soo, have one schedule class for ECF and another schedule class for NPQ? 

Ehh, I am not sure I like it, cause `ParticipantProfiles` would need to point at two different schedule models. And, well, the underlying structure is "a schedule has an identifier and a list of milestones" for both ECF and NPQ. And I can't think of a meaningful behaviour difference. 

The only thing I can think of is that when we are changing schedules we don't want ecf participants to get NPQ schedules and vice versa, but I think scopes are a more appropriate tool to handle that than namespaces.